script.js

```diff
@@ -0,0 +1,201 @@
+// ====== html 요소 변수 선언  ======= //
+
+const form = document.querySelector(".formControl");
+const formInput = document.querySelector(".formInput");
+const submitBtn = document.querySelector(".submitBtn");
+const todoList = document.querySelector(".todoList");
+const doneList = document.querySelector(".doneList");
+const todoCountField = document.querySelector(".todoCount");
+const doneCountField = document.querySelector(".doneCount");
+
+// form | form 요소
+// formInput | form 입력 영역 요소
+// submitBtn | form 내 등록 버튼
+// todoList | todoItem 가지는 list container
+// doneList | doneItem 가지는 list container
+// todoCountField, doneCountField | 각 count 값 가지는 text container
+
+// ====== 초기화시 실행 동작 ======= //
+
+window.addEventListener("DOMContentLoaded", () => {
+  updateCnt();
+  let items = getLocalStorage();
```

Comment:
`items`가 꼭 `let`이여아 할 필요가 있을까요? 부득이한 경우를 제외하고는 `const`를 사용하는 것이 일반적입니다.
```suggestion
  const items = getLocalStorage();
```

Reply:
맞습니다. 참고하겠습니다!